text/0000-bionic-mixins.md

```diff
@@ -0,0 +1,75 @@
+# Meta
+[meta]: #meta
+- Name: Defining Mixins for io.buildpacks.stacks.bionic
+- Start Date: 2019-12-05
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+The Buildpack API Specification allows stack maintainers to define stack mixins for a given stack.
+The Cloud Native Buildpacks project informally defines stack build and run images with ID `io.buildpacks.stacks.bionic` as containing exactly the set of Ubuntu packages in the `ubuntu:bionic` image on Docker Hub.
+This RFC proposes that we formalize the existing definition of `io.buildpacks.stacks.bionic` and define mixins on that stack to be Ubuntu 18.04 LTS package names (without version or architecture info) from official package mirrors. 
+
+In general, a stack is a contract that should be more like "Ubuntu Bionic" and less like "Platform XYZ's Re-distribution of Ubuntu Binoic."
+Stack images can still be provided by platforms/vendors, but they should use the standard CNB stack contracts where applicable.
+
+# Motivation
+[motivation]: #motivation
+
+This will allow:
+1. Buildpack authors to publish buildpacks that are compatible with more stack images without requiring those authors to explicitly specify vendor-specific stack IDs.
```

Comment:
How can a platform specify that their custom stack adheres to `io.buildpacks.stacks.bionic`?  (How would `heroku/buildpacks:18` transition, considering there are Buildpacks in the wild relying on the `heroku-18` stack ID?)

Reply:
Platforms could publish new images with the `io.buildpacks.stacks.bionic` ID  to new image tags and deprecate their bespoke stack image tags once everyone has migrated.

Alternatively, platforms could give buildpack authors a certain amount of time to add both stacks to their buildpack.toml files before publishing images with the `io.buildpacks.stacks.bionic` ID over the existing image tags.